apps/src/applab/designElements/elementUtils.js

```diff
@@ -220,10 +220,70 @@ export function setDefaultBorderStyles(element, options = {}) {
   }
   if (forceDefaults || element.style.borderColor === '') {
     element.style.borderColor = textInput
-      ? color.text_input_default_border_color
-      : color.black;
+      ? themeColor.textInputBorder.classic
+      : themeColor.dropdownBorder.classic;
   }
   if (forceDefaults || element.style.borderRadius === '') {
     element.style.borderRadius = '0px';
   }
 }
+
+/**
+ * Parse a padding string and return the total horizontal padding and
+ * total vertical padding.
+ * @param {string} cssPaddingString value from element.style.padding
+ */
+export function calculatePadding(cssPaddingString) {
+  // Extract up to 4 numbers, ignoring the 'px' that may be included
+
+  // NOTE: if other measurement values (e.g. 'em') make it into the padding string,
+  // we will treat them as 'px' values
+
+  // Ideally, we could use getComputedStyle(), but we need to work with
+  // detached DOM nodes, which doesn't work on Chrome and Safari
+
+  const paddingValues = (cssPaddingString || '')
+    .split(/\s+/)
+    .map(part => parseInt(part, 10));
+  for (
+    var validPaddingValues = 0;
```

Comment:
Should be validPaddingValueCount or something similar. The current name makes it sound like this is a list.